MELA/interface/Mela.h

```diff
@@ -199,7 +205,8 @@ class Mela{
     );
   void computeP(
     float& prob,
-    bool useConstant=true
+    bool useConstant=true,
+    int nhel=-1
```

Comment:
nhel is added as a default parameter (-1 implies a sum over all helicities a la JHUGenMELA) in order to allow for someone to choose a helicity value (as you are able to do so in Madgraph reweighting). If madgraph reweighting is not used, nhel will never be accessed.

Reply:
I would prefer to have that as a property of the particle, i.e. an additional property in MELAParticle, which would get inherited by all daughter classes. Moreover, helicities are particle properties, not a general total matrix element property. To also add, -1 could be confusing as 'down' - it might be better to assign -99 as a default. To avoid consuming unnecessary memory, it is best to keep the type minimal ([signed] char).

Reply:
The helicity isn't actually a property of the particle in this case - it's an input to the reweighting function for Madgraph. The `smatrixhel` function takes in a list of pdgs, process IDs, pdg numbers, 4 vectors, an alphaS value, a scale2 value (irrelevant here), the helicity number (-1 if summing over all helicities), and the answer.